cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -73,11 +80,21 @@ DrawableSubmap::DrawableSubmap(const int trajectory_id, const int submap_index,
   material_->setCullingMode(Ogre::CULL_NONE);
   material_->setDepthBias(-1.f, 0.f);
   material_->setDepthWriteEnabled(false);
-  scene_node_->attachObject(manual_object_);
+  visibility_ = cartographer::common::make_unique<::rviz::BoolProperty>(
```

Comment:
Yes, we're not always perfect :) However, within this file I believe it's always ::cartographer.